test/rules/EC2.test.ts

```diff
@@ -784,20 +788,189 @@ describe('Amazon Elastic Compute Cloud (Amazon EC2)', () => {
 describe('Amazon Elastic Block Store (EBS)', () => {
   describe('EC2EBSVolumeEncrypted: EBS volumes have encryption enabled', () => {
     const ruleId = 'EC2EBSVolumeEncrypted';
-    test('Noncompliance 1', () => {
+    test('Noncompliance 1 - Volume', () => {
       new Volume(stack, 'rVolume', {
         availabilityZone: stack.availabilityZones[0],
         size: Size.gibibytes(42),
         encrypted: false,
       });
       validateStack(stack, ruleId, TestType.NON_COMPLIANCE);
     });
-    test('Compliance', () => {
-      new Volume(stack, 'rVolume', {
+    test('Noncompliance 2 - CfnVolume', () => {
+      new CfnVolume(stack, 'rVolume', {
```

Comment:
Let's remove the `r` from the Ids ( ex. `rVolume` -> `Volume`). It's from a naming convention used for regular CloudFormation that I initially carried over to here but am removing as new PRs are made.